### DISCUSSION TITLE
Communication Medium

### DISCUSSION THREAD

**Discussion by mindoodoo** [ratatui/ratatui#65] [Discussion]
# Update

For the time being, we have settled on using discord (same server indicated below). Soon enough we will implement a matrix bridge for contributors that may prefer that option.

# Communication Medium

Hey guys, let's get everyone's thoughts on the communication medium we use in addition to github. For the time being we've been using [discord](https://discord.gg/pMCEU9hNEj), but this has only been temporarily for the time being.

### Some options that have been mentioned :

- Zulip
- Discord
- Matrix
- Irc
- use of bridges to use multiple platforms

**Comment by tranzystorekk** [comment]
Some prominent examples of Matrix communities:

- #rust:matrix.org - rustlang
- #cross-rs:matrix.org - cross tool
- #forgejo-chat:matrix.org / #codeberg.org:matrix.org - Codeberg platform and the adjacent Forgejo software forge

**Comment by AntoniosBarotsis** [comment]
Rust also uses both discord and zulip as well.
